vignettes/ChromBand.Rmd

```diff
@@ -0,0 +1,379 @@
+
+<a href="#fig:chr12ideogram">Figure 1</a> shows an ideogram of human chromosome 12, with the band 12q21 shaded. As shown in the figure, 12q21 can be divided into more granular levels 12q21.1, 12q21.2, and 12q21.3. 12q21.3 can itself be divided at an even finer level of resolution into 12q21.31, 12q21.32, and 12q21.33. Moving towards less granular bands, 12q21 is a part of 12q2 which is again a part of 12q. We take advantage of this nested structure of the bands in our analysis.
+
+```{r Category, message=FALSE}
+library("Category") 
+library("ALL") 
+library("hgu95av2.db")
+library("annotate") 
+library("genefilter") 
+##library("SNPchip")
+library("karyoploteR") 
+library("geneplotter") 
+library("limma") 
+library("lattice")
+library("graph")
+```
+
+# Data Preparation
+
+For illustration, we use a microarray dataset [@Chiaretti2005] from a clinical trial in acute lymphoblastic leukemia (ALL). The data are described in Chapter 2 of [@BiocCaseStudies].
+
+<figure id="fig:chr12ideogram">
+
+```{r chr12ideogram, echo=FALSE, fig.align="center", fig.width=9, fig.height=3}
+
+build <- "hg19" ## or "hg18"
+kp <- plotKaryotype(genome=build, chromosomes="chr12", main="Human chromosome12" )
+## cyt2 <- getCytoband(build=build)
+## cyt2$gieStain <- "foo"
+## c12p12_idx <- intersect(grep("^q21", cyt2$name),
+##                         which(cyt2$chrom == "12"))
+## cyt2[c12p12_idx, "gieStain"] <- rep(c("gpos50", "gpos75"),
+##                                     length=length(c12p12_idx))
+##
+##
+## plotCytoband2(chromosome="12", build=build, cytoband=cyt2, outer=FALSE,  ## From SNPchip
+##     cex.axis=0.6, main="Human chromosome 12")
+## plotCytoband2(chromosome="12", build=build, cytoband=cyt2, outer=FALSE,
+##               cex.axis=0.6,
+##               main="Human chromosome 12")
+```
+
+<figcaption>Figure 1: Ideogram for human chromosome 12. The p arm is on the left, the q arm is on the right, and the centromere is indicated by a notch. The shaded bands together represent 12q21. This band is composed of three sub-bands: 12q21.1, 12q21.2, and 12q21.3. The last of these is composed of sub-sub-bands 12q21.31, 12q21.32, and 12q21.33.</figcaption>
+
+</figure>
+
+For the analysis presented here, we consider the comparison of two subsets: patients identified as having a BCR/ABL gene fusion present, typically as a result of a translocation of chromosomes 9 and 22 (labeled BCR/ABL), and those that have no observed cytogenetic abnormalities (labeled NEG). The full dataset is available in the [ALL]{.sans-serif} package, and the relevant subset of the data can be obtained by
```

Comment:
Replace `[ALL]{.sans-serif}` with the canonical link for the Bioconductor package ALL using the macro described in BiocStyle at https://bioconductor.org/packages/release/bioc/vignettes/BiocStyle/inst/doc/AuthoringRmdVignettes.html#4_Style_macros. Do this for all Bioconductor packages in the remainder of the vignette.

Reply:
ok

Reply:
\@ref  was what I was up all night trying to fid solution , it never worked , I looked everywhere for the solution including this documentations  you gave it never worked,I only used the one that worked. Can you just show me the code syntax. 
In my understanding the \@ref code is a latex (.tex) syntax for referencing, Kindly show me how to use it in code. And I used html since the Rmd document can render Html.